app/components/app_session_overview_tallies_component.rb

```diff
@@ -41,21 +52,64 @@ def tally_cards_for_programme(programme)
             vaccination_status: "could_not_vaccinate",
             programme_types: [programme.type]
           )
-      },
-      {
-        heading: "No outcome",
-        colour: "grey",
-        count: no_outcome_count(programme).to_s,
-        link_to:
-          session_patients_path(
-            session,
-            vaccination_status: "none_yet",
-            programme_types: [programme.type]
-          )
       }
     ]
   end
 
+  def consent_cards
+    cards =
+      programmes.flat_map do |programme|
+        if programme.has_multiple_vaccine_methods?
+          [
```

Comment:
Rather than hard coding the vaccine methods, I think we should be able to iterate over `Programme#vaccine_methods`, and there should be locale strings available to get their human readable name.